src/components/section-list.component.js

```diff
@@ -15,42 +16,49 @@ type Props = {
   buttonAction: Function,
 };
 
-const styles = StyleSheet.create({
-  section: {
-    marginTop: 15,
-  },
-  topHeader: {
-    flex: 1,
-    flexDirection: 'row',
-    justifyContent: 'space-between',
-    alignItems: 'center',
-  },
-  sectionTitle: {
-    color: colors.black,
-    padding: 15,
-    ...fonts.fontPrimaryBold,
+const Section = styled.View`
+  margin-top: 15;
+`;
+const Header = styled.View`
+  flex: auto;
```

Comment:
Does `flex: auto` the same with `flex: 1`?

Reply:
I couldn't get flex: 1 to work it would result in the content below overlapping the header.

Reply:
Hmm, the original code uses `flex: 1`. Never mind, maybe you discovered a bug and fixed. We can help you check it.

Reply:
This is what I get with `flex: 1`. 
![simulator screen shot - iphone 6 - 2018-02-22 at 22 20 13](https://user-images.githubusercontent.com/3621147/36538300-9be10206-181e-11e8-8c23-15615282c553.png)

`flex: 0` seemed to fix the flexbox issues.
![simulator screen shot - iphone 6 - 2018-02-22 at 22 21 30](https://user-images.githubusercontent.com/3621147/36538413-10f191d2-181f-11e8-9c09-a4ff5ef18e36.png)

Also `flex-grow: 0; flex-basis: 0;` also looked good.
![simulator screen shot - iphone 6 - 2018-02-22 at 22 38 51](https://user-images.githubusercontent.com/3621147/36539051-3097b762-1821-11e8-9177-5a3290d41090.png)

I've gone with `flex-grow: 0; flex-basis: 0;`

Reply:
@jamesg1 Interesting. `flex: 1` of styled-components really results in weird UI, while the original code works well.

But the initial values are `flex-grow: 0; flex-shrink: 1; flex-basis: auto;`. You can delete `flex-grow: 0; flex-basis: 0;`.

Reply:
I deleted `flex-grow: 0; flex-basis: 0;` on Header styling.
Thanks